flipt-evaluation/src/models/common.rs

```diff
@@ -40,6 +40,8 @@ pub enum ConstraintComparisonType {
     Boolean,
     #[serde(rename = "DATETIME_CONSTRAINT_COMPARISON_TYPE")]
     DateTime,
+    #[serde(rename = "ENTITY_ID_CONSTRAINT_COMPARISON_TYPE")]
+    EntityId,
```

Comment:
I think we can make `Unknown` the fallback value by moving it to the last variant of this enum and tagging it like so:

```rust
#[derive(Clone, Debug, Default, Deserialize, PartialEq)]
pub enum ConstraintComparisonType {
    #[serde(rename = "STRING_CONSTRAINT_COMPARISON_TYPE")]
    String,
    #[serde(rename = "NUMBER_CONSTRAINT_COMPARISON_TYPE")]
    Number,
    #[serde(rename = "BOOLEAN_CONSTRAINT_COMPARISON_TYPE")]
    Boolean,
    #[serde(rename = "DATETIME_CONSTRAINT_COMPARISON_TYPE")]
    DateTime,
    #[serde(rename = "ENTITY_ID_CONSTRAINT_COMPARISON_TYPE")]
    EntityId,
    #[default]
    #[serde(other)]
    #[serde(rename = "UNKNOWN_CONSTRAINT_COMPARISON_TYPE")]
    Unknown,
}
```

https://serde.rs/variant-attrs.html#other

Reply:
we should likely do this for all enum values that we can to prevent the panic errors on future additions

Reply:
@markphelps Done!

Reply:
@yquansah can you add a test to show that it successfully deserializes to `Unknown` given a value not in this enum in JSON pls